repo_policy_compliance/check.py

```diff
@@ -228,30 +228,25 @@ def collaborators(github_client: Github, repository_name: str) -> Report:
     return Report(result=Result.PASS, reason=None)
 
 
-def _branch_external_fork(repository: Repository, source_repository_name: str) -> bool:
-    """Check whether a branch is an external fork.
-
-    A external fork is a fork that is not owned by a user who has push or above permission on the
-    repository.
+@log.call
+def disallow_fork(
```

Comment:
We should probably check whether it is a fork from a Canonical employee, we might need to rename the rule to be clearer. The check for if it isn't a fork is good to have first to reduce API calls

Reply:
Updated, thank you!